collector/snapshots.go

```diff
@@ -183,6 +183,24 @@ func NewSnapshots(logger log.Logger, client *http.Client, url *url.URL) *Snapsho
 				},
 				Labels: defaultSnapshotRepositoryLabelValues,
 			},
+			{
+				Type: prometheus.GaugeValue,
+				Desc: prometheus.NewDesc(
+					prometheus.BuildFQName(namespace, "snapshot_stats", "latest_snapshot_timestamp"),
```

Comment:
It'd be great to follow Prometheus best practices and include the metric unit in description and metric name, like "Timestamp of the latest SUCCESS or PARTIAL snapshot in seconds" and `...latest_snapshot_timestamp_seconds`. See https://prometheus.io/docs/practices/naming/

Reply:
agreed, I named it like this because the other metric is `oldest_snapshot_timestamp` already, so I guess its best to have all metrics here in the same pattern ? 🤔

anyway, I'm happy to change it if maintainers think we should...

Reply:
yes, please change the metric names so they follow the best practise. I'll create a new release after the merge and add a note to the release notes about the breakting change.

Thanks for your contribution and sorry I'm a bit slow in reacting to PR and issues here

Reply:
```suggestion
					prometheus.BuildFQName(namespace, "snapshot_stats", "latest_snapshot_seconds"),
```